.gitignore

```diff
@@ -1,3 +1,7 @@
 bin/
 .gopath/
-vendor/github.com/gdm85/go-libdeluge
+vendor/
+
+# misc
+/.idea/
+/.vscode/
```

Comment:
Please remove this, see https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
done.

should match up with master now.